org.KangLin.Rabbit.RemoteControl.json

```diff
@@ -0,0 +1,262 @@
+/* See: https://docs.flatpak.org/en/latest/manifests.html */
+/* See: https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest */
```

Comment:
```suggestion
```

it's like a comment saying water is wet.

Reply:
It's easy for me to review the documentation :)

Reply:
no. it's noise.

Reply:
not resolved